src/SignalR/clients/csharp/Client/test/UnitTests/HubConnectionTests.Reconnect.cs

```diff
@@ -17,896 +18,899 @@ namespace Microsoft.AspNetCore.SignalR.Client.Tests
 {
     public partial class HubConnectionTests
     {
-        [Fact]
-        public async Task ReconnectIsNotEnabledByDefault()
+        public class Reconnect : VerifiableLoggedTest
```

Comment:
Is this the only change to this file?

Reply:
https://github.com/aspnet/AspNetCore/pull/10433/commits/5a97797b0112ba2f6b5bf45a047e0964c2cb4493#diff-071f9acf91eca9ec3e4b318ccfea3037 and removing "Reconnect" from the beginning of the test methods are the only other changes in the file. If you add "?w=1" to the URL or click the "No Whitespace" button.